openedx/core/djangoapps/content/course_overviews/serializers.py

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
+"""
+CourseOverview serializers
+"""
+from rest_framework import serializers
+
+from openedx.core.djangoapps.content.course_overviews.models import CourseOverview
+
+
+class CourseOverviewBaseSerializer(serializers.ModelSerializer):
+    """
+    Serializer for a course run overview.
+    """
+
+    class Meta(object):
+        model = CourseOverview
+        fields = '__all__'
+
+    def to_representation(self, instance):
+        representation = super(CourseOverviewBaseSerializer, self).to_representation(instance)
+        representation['display_name_with_default'] = instance.display_name_with_default
+        representation['has_started'] = instance.has_started
+        representation['has_ended'] = instance.has_ended
+        representation['pacing'] = instance.pacing
```

Comment:
Did you have to do this because these are properties of CourseOverview and not attributes?

Reply:
yup

Reply:
as with anything django, there are other ways of adding fields to the serializer that are not on the model, but i think this is most readable